src/client/app/app.react.js

```diff
@@ -94,7 +96,7 @@ class App extends Component {
         <RouteHandler {...this.state} />
         <footer>
           <p>
-            made by <a href="https://twitter.com/steida">steida</a>
+            <FormattedHTMLMessage message={msg('app.madeByHtml')} />
```

Comment:
I do like Markdown myself for BFU publishing, still I favour HTML:
- about 30% faster
- much more flexible
- one less abstraction layer
And we may end up trying to figure out using custom React components in translations without unnecessarily splitting texts (that will be the real challenge).